### DISCUSSION TITLE
OpenMCT Architecture Documentation: Will this part be updated?

### DISCUSSION THREAD

**Discussion by marcelo-earth** [nasa/openmct#5268] [Community Q&A] [answered]
Hi OpenMCT contributors! I'm learning about OpenMCT, and I found this [document from NASA](https://nasa.github.io/openmct/docs/architecture/framework.html) on the web. But this document makes references to Angular, and since Angular is not used anymore, I assume this is for the previous version.

**Will these documents be updated? Have the architectures changed so much from that version?**

So many thanks in advance! 👋 

## Screenshots
<img width="1243" alt="OpenMCT Docs" src="https://user-images.githubusercontent.com/47821093/171047041-6779d72d-dcee-40de-b413-33e1a6f5624d.png">
<img width="1042" alt="OpenMCT Docs" src="https://user-images.githubusercontent.com/47821093/171047755-e6d6ce92-74ae-42ad-b23a-ff512dcae485.png">

**Comment by MarkCodering** [comment, marked as answer]
@360macky I think this documentation has already outdated. Most of the documentation available for NASA Open MCT is shown here [NASA Open MCT API Documentation](https://github.com/nasa/openmct/blob/master/API.md). Will be interested in creating new beginner-friendly documentation or tutorial for non-CS students or departments to promote the use of NASA Open MCT? 
Reference: #5336

**Comment by marcelo-earth** [comment]
Hi @MarkCodering!

Thanks. I've been reviewing that documentation of the API. To understand how OpenMCT works internally, I began to create some graphic schemes for myself. I still have more sections to explore. But I'm on it.

The first ones I drew:
<img width="1439" alt="image" src="https://user-images.githubusercontent.com/47821093/173212733-2bed8d31-7982-4c15-824b-5a946e3e37ea.png">

Yes, I'm very interested in creating a new beginner-friendly documentation for Open MCT. Sounds great!

I just read your poll, and I read that you plan to include "graphic presentation to explain abstract concepts in the framework". I'm very supportive of that.

**Reply by MarkCodering** [reply]
@360macky A great diagram to share! I am looking forward to collaborate with you on this project, #5336. Do you have any suggestion on the plan for creating a beginner-friendly documentation for new-comers in this framework?

**Reply by marcelo-earth** [reply]
That's awesome!

I would like to include a [roadmap.sh style graphic](https://roadmap.sh/frontend).
In this way we provide the most basic concepts to the most advanced.

For the graphics I have used [Excalidraw](http://excalidraw.com). Do you have any other options to create graphics?

And, along with a roadmap, tutorials with diagrams of how certain mechanisms work within OpenMCT.

What do you think? What do you suggest?
